draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2123,16 +2123,31 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
    mechanism and configuration techniques as general-purpose Web browsers.
 </t>
 <t>
-   HTTP communication is initiated by a user agent for some purpose.
-   The purpose is a combination of request semantics
-   and a target resource upon which to apply those
-   semantics.  A URI reference (<xref target="uri"/>) is typically used as
-   an identifier for the "<x:dfn>target resource</x:dfn>", which a user agent
-   would resolve to its absolute form in order to obtain the
-   "<x:dfn>target URI</x:dfn>".  The target URI
-   excludes the reference's fragment component, if any,
-   since fragment identifiers are reserved for client-side processing
-   (<xref target="RFC3986" x:fmt="," x:sec="3.5"/>).
+   HTTP communication is initiated by a user agent for some purpose. The
+   purpose is a combination of request semantics and a target resource upon
+   which to apply those semantics. The "<x:dfn>request target</x:dfn>" is
+   the protocol element that identifies the "<x:dfn>target resource</x:dfn>".
+</t>
+<t>
+   Typically, the request target is a URI reference (<xref target="uri"/>)
+   which a user agent would resolve to its absolute form in order to obtain
+   the "<x:dfn>target URI</x:dfn>". The target URI excludes the reference's
+   fragment component, if any, since fragment identifiers are reserved for
+   client-side processing (<xref target="RFC3986" x:fmt="," x:sec="3.5"/>).
+</t>
+<t>
+   However, there are two special, method-specific forms allowed for the
+   request target in specific circumstances:
+</t>
+<ul>   
+<li>For CONNECT (<xref target="CONNECT"/>), the request target is the host
+name and port number of the tunnel destination, separated by a colon.</li>
+<li>For OPTIONS (<xref target="OPTIONS"/>), the request target can be a
+single asterisk ("*").</li>
+</ul>
```

Comment:
(indentation)

Reply:
also maybe refer to #asterisk-form in messaging?

Reply:
Indentation fixed. WRT #asterisk-form, I'd be more inclined to move most of that into Semantics. Thoughts?

Reply:
Yes, we probably need that in semantics.